benchmarks/comparison.py

```diff
@@ -0,0 +1,39 @@
+import json
+import os
+import shutil
+import subprocess
+import tempfile
+
+from django.conf import settings
+
+compare_script = os.path.join(os.path.dirname(__file__), '../ext/art-testing/compare.py')
+
+def render_comparison(results_before, results_after):
+    tmpdir = export_data(results_before, results_after)
+    # FIXME: sandbox this!
+    output = subprocess.check_output([
```

Comment:
how about importing the function from compare.py and running it natively? It might not work right now as compare.py depends on python3. Should we migrate to python3 as well?